### ISSUE TITLE
Link to papers are no longer valid

### ISSUE THREAD

**Issue by TheProjectsGuy** [ptrbrtz/razor-9dof-ahrs#67]
The links to the DCM papers (in [this](https://github.com/Razor-AHRS/razor-9dof-ahrs/wiki/Tutorial#sensor-fusion-algorithm) section of the wiki) are no longer valid. Kindly update them

**Comment by gauxonz** [comment]
Yes. Please update the paper links. Also, could you make this repo (alone with the WIKI) citable? The guide is [here](https://guides.github.com/activities/citable-code/)
